Author: Milan P. StaniÄ‡ <mps@arvanta.net>
Date:   2021-06-28 18:49:49 +0000

set more 'natural' order of e/mmc devices, internal emmc is mmcblk0

--- a/arch/arm64/boot/dts/rockchip/rk3399-gru.dtsi	2021-06-27 22:21:11.000000000 +0000
+++ b/arch/arm64/boot/dts/rockchip/rk3399-gru.dtsi	2021-06-28 18:49:49.370552503 +0000
@@ -11,8 +11,8 @@
 
 / {
 	aliases {
-		mmc0 = &sdmmc;
-		mmc1 = &sdhci;
+		mmc1 = &sdmmc;
+		mmc0 = &sdhci;
 	};
 
 	chosen {
